app/forms/state_file/name_dob_form.rb

```diff
@@ -3,6 +3,8 @@ class NameDobForm < QuestionsForm
     include DateHelper
 
     attr_accessor :dependents_attributes
+    NAME_VALIDATION_MSG = I18n.t('state_file.questions.name_dob.edit.validation_name').freeze
+    FIRST_NAME_VALIDATION_MSG = NAME_VALIDATION_MSG + I18n.t('state_file.questions.name_dob.edit.validation_first_name').freeze
 
```

Comment:
Looked over this with Em and we think we should format the error messages in yaml as separate phrases instead of combining/formatting it in the form.

Reply:
Done